src/templates/dso_api/dynamic_api/api.html

```diff
@@ -63,8 +67,458 @@
+            {% with form=raw_data_post_form %}
+              <form action="{{ request.get_full_path }}" method="POST" class="form-horizontal">
+                <fieldset>
+                  {% include "rest_framework/raw_data_form.html" %}
+                  <div class="form-actions">
+                    <button class="btn btn-primary js-tooltip" title="Make a POST request on the {{ name }} resource">POST</button>
+                  </div>
+                </fieldset>
+              </form>
+            {% endwith %}
+          </div>
+        </div>
+      </div>
+    {% endif %}
+
+    {% if put_form or raw_data_put_form or raw_data_patch_form %}
+      <div {% if put_form %}class="tabbable"{% endif %}>
+        {% if put_form %}
+          <ul class="nav nav-tabs form-switcher">
+            <li>
+              <a name='html-tab' href="#put-object-form" data-toggle="tab">HTML form</a>
+            </li>
+            <li>
+              <a  name='raw-tab' href="#put-generic-content-form" data-toggle="tab">Raw data</a>
+            </li>
+          </ul>
+        {% endif %}
+
+        <div class="well tab-content">
+          {% if put_form %}
+            <div class="tab-pane" id="put-object-form">
+              <form action="{{ request.get_full_path }}" data-method="PUT" enctype="multipart/form-data" class="form-horizontal" novalidate>
+                <fieldset>
+                  {{ put_form }}
+                  <div class="form-actions">
+                    <button class="btn btn-primary js-tooltip" title="Make a PUT request on the {{ name }} resource">PUT</button>
+                  </div>
+                </fieldset>
+              </form>
+            </div>
+          {% endif %}
+
+          <div {% if put_form %}class="tab-pane"{% endif %} id="put-generic-content-form">
+            {% with form=raw_data_put_or_patch_form %}
+              <form action="{{ request.get_full_path }}" data-method="PUT" class="form-horizontal">
+                <fieldset>
+                  {% include "rest_framework/raw_data_form.html" %}
+                  <div class="form-actions">
+                    {% if raw_data_put_form %}
+                      <button class="btn btn-primary js-tooltip" title="Make a PUT request on the {{ name }} resource">PUT</button>
+                    {% endif %}
+                    {% if raw_data_patch_form %}
+                    <button data-method="PATCH" class="btn btn-primary js-tooltip" title="Make a PATCH request on the {{ name }} resource">PATCH</button>
+                      {% endif %}
+                  </div>
+                </fieldset>
+              </form>
+            {% endwith %}
+          </div>
+        </div>
+      </div>
+    {% endif %}
+  {% endif %}
+{% endblock content %}
+
+{% block script %}
+<script>
+  window.drf = {
+    csrfHeaderName: "{{ csrf_header_name|default:'X-CSRFToken' }}",
+    csrfToken: "{% if request %}{{ csrf_token }}{% endif %}"
+  };
+</script>
+<script>
```

Comment:
Omdat hier zoveel gebeurt zou ik om de cognitieve load te verlagen aan het begin van de file even een `{% comment %} .... {% endcomment %}` sectie opnemen waarin je uitlegd waarom deze template bestaat. Dat:

* vanwege het ontbreken van de page count er pas client-side tijdens het renderen paginering bekend is.
* de response via `fetch()` streaming ingeladen wordt i.p.v. vast ingesloten in de template, wat bovendien een DOS-aanvalsvector zou opleveren bij grote datasets.